### PR TITLE
Hotfix | Show correct DAO name on Node Page.

### PR DESCRIPTION
**Pull request by Da-Colon**
## Screenshots (if applicable)
localhost_3000_(Desktop) (27)

### CODE REVIEW

File: src/components/ui/cards/DAOInfoCard.tsx [hunk 1]
```diff
@@ -27,12 +28,18 @@ export function DAOInfoCard({
   numberOfChildrenDAO,
 }: IDAOInfoCard) {
   const {
-    gnosis: { daoName },
+    gnosis: {
+      safe: { address },
+      daoName,
```
Comment (herbig) [inline src/components/ui/cards/DAOInfoCard.tsx:33]: what is this name? If it's not the DAO registry name..

File: src/components/ui/cards/DAOInfoCard.tsx [hunk 2]
```diff
@@ -27,12 +28,18 @@ export function DAOInfoCard({
...
+    },
     account: {
       favorites: { favoritesList, toggleFavorite },
     },
   } = useFractal();
   const copyToClipboard = useCopyText();
+  const { daoRegistryName } = useDAOName({
+    address: address !== safeAddress ? safeAddress : undefined,
```
Comment (mobiusTripper-crypto) [inline src/components/ui/cards/DAOInfoCard.tsx:41]: it seems like we should only be retrieving the daoName from one place, maybe 
```
  const { daoRegistryName } = useDAOName({
    address: address !== safeAddress ? safeAddress : address,
  });
```
and stop getting if from the gnosis object? (I have no idea what other consequences that may cause)

**Review by mobiusTripper-crypto — APPROVED** [review]
a comment was left

File: src/components/ui/cards/DAOInfoCard.tsx (hunk 1 above)
Comment (Da-Colon) [reply src/components/ui/cards/DAOInfoCard.tsx:33]: undefined I believe

File: src/components/ui/cards/DAOInfoCard.tsx (hunk 2 above)
Comment (Da-Colon) [reply src/components/ui/cards/DAOInfoCard.tsx:41]: I guess the thought here is to save on duplicate calls as renders happen.

File: src/components/ui/cards/DAOInfoCard.tsx (hunk 1 above)
Comment (Da-Colon) [reply src/components/ui/cards/DAOInfoCard.tsx:33]: is that what you meant?

Comment (Da-Colon) [reply src/components/ui/cards/DAOInfoCard.tsx:33]: ah its not. useDAOName is used when its retrieved. so it goes through the same ens -> DAO name -> substring

**Review by herbig — APPROVED** [review]

**Pull request merged**